### ISSUE TITLE
Canvases need text equivalent (severity 6)

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1623]
This is a Mirador issue ultimately, but I'm starting by recording it here, because it's reported here:
"This canvas element does not have a mechanism that allows an accessible name value to be calculated." AMP gives it a severity level of 6/10.

Placeholder for discussion, as I don't know the best way to solve it.

**Comment by jkeck** [comment]
A ticket will be created in Mirador, and handled in the next work cycle focused on that.
